server/src/modules/dispositif/dispositif.repository.ts

```diff
@@ -79,7 +79,7 @@ export const getDispositifById = async (
   id: DispositifId,
   neededFields: DispositifFieldsRequest = {},
   populate: string = ""
-): Promise<Dispositif> => DispositifModel.findOne({ _id: id }, neededFields).populate(populate);
+) => DispositifModel.findOne({ _id: id }, neededFields).populate(populate);
```

Comment:
J'ai vu que la fonction `findById` était préconisée maintenant.

```suggestion
) => DispositifModel.findById(id, neededFields).populate(populate);
```